content/en/blog/_posts/2022-12-06-kubernetes-1.25-blog.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: blog
+title: "Kubernetes v1.26: Electrifying"
+date: 2022-12-6
+slug: kubernetes-v1-26-release
+---
+
+**Authors**: [Kubernetes 1.26 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.26/release-team.md)
+
+It's with immense joy that we announce the release of Kubernetes v1.26!
+
+This release includes a total of 37 enhancements: eleven of them are graduating to Stable, ten are graduating to Beta, and sixteen of them are entering Alpha. We also have two features being deprecated or removed.
+
+## Release theme and logo 
+
+**Kubernetes 1.26: Electrifying**
+
+\<image here>
+
+The theme for Kubernetes v1.26 is _Electrifying_.
+
+Each Kubernetes release is the result of the coordinated effort of dedicated volunteers, and only made possible due to the use of a diverse and complex set of computing resources, spread out through multiple datacenters and regions worldwide. The end result of a release - the binaries, the image containers, the documentation - are then deployed on a growing number of personal, on-premises, and cloud computing resources. 
```

Comment:
...and only made possible due to the use of a diverse and complex set of computing resources...

IMO reads a bit like contributors are machines, lol

(feel free to rephrase my suggestion :D) 
```suggestion
Each Kubernetes release is the result of the coordinated effort of dedicated volunteers, supported by complex automation processes and tools, powered by a vast set of computing resources spread across multiple data centers and regions worldwide. The end result of this open source process are after each release - the Kubernetes binaries, container images, and documentation - which are then being used on a growing number of personal, on-premises, and cloud computing resources.
```

Reply:
Ok, got your point, but OTOH I don't thing the suggested alternative really "flows"... what about this (I've removed some words which are not really needed to make

> Each Kubernetes release is the result of the coordinated effort of dedicated volunteers, supported by complex automation processes and tools that are powered by a vast set of computing resources spread across multiple data centers worldwide. The result of this open source process - the Kubernetes binaries, container images, documentation, and others - is then used on a growing number of personal, on-premises, and cloud computing resources.

Reply:
👍